### ISSUE TITLE
large schema filesize will error out as getconf ARG_MAX 1048576

### ISSUE THREAD

**Issue by GandiYaswanthSai** [wundergraph/wundergraph#1011]
### Bug description

I am using a large prisma file(500 models approximately) for development and I am unable to do the introspection. It is throwing error with arg too long.
`{"level":"error","time":1685103624369,"pid":2253,"hostname":"9477e6c110a5","component":"@wundergraph/sdk","msg":"database introspection failed: Error: Command failed with exit code 1: /app/wundergraph-sample/node_modules/@wundergraph/wunderctl/download/wunderctl introspect prisma prisma/schema.prisma --outfile=generated/introspection/database/1574e0d9ea6c0dbcd2c502ec7d4961f67fd31.json --debug --wundergraph-dir /app/wundergraph-sample/src --cli-log-level info --pretty-logging=false\nError: fork/exec /root/.cache/wundergraph/prisma/efdf9b1183dddfd4258cd181a72125755215ab7b/prisma-query-engine-debian-openssl-1.1.x: argument list too long\n{\"level\":\"debug\",\"time\":1685103624353,\"caller\":\"commands/root.go:93\",\"msg\":\"starting without env file\",\"hostname\":\"9477e6c110a5\",\"pid\":2331,\"component\":\"@wundergraph/wunderctl\"}"}`

### How to reproduce

1. create new nextjs-postgres-prisma example
2. Add Database URL
3. Add as many Models as possible.
4. Run ''yarn wunderctl up --debug''
5. See error:
`Error: fork/exec /root/.cache/wundergraph/prisma/efdf9b1183dddfd4258cd181a72125755215ab7b/prisma-query-engine-debian-openssl-1.1.x: argument list too long\n{\"level\":\"debug\",\"time\":1685103624353,\"caller\":\"commands/root.go:93\",\"msg\":\"starting without env file\",\"hostname\":\"9477e6c110a5\",\"pid\":2331,\"component\":\"@wundergraph/wunderctl\"}`
File:
[schema.zip](https://github.com/wundergraph/wundergraph/files/11591024/schema.zip)

### Expected behavior

`{"level":"info","time":1685345418235,"pid":162,"hostname":"df8a84785b33","component":"@wundergraph/sdk","msg":"Build completed."}`

### WunderGraph information

wundergraph.config.ts file
```
import {introspect, configureWunderGraphApplication, Api} from '@wundergraph/sdk';


const api = introspect.prisma({
    apiNamespace: "tenantName",
    prismaFilePath: "./prisma/schema.prisma",
    introspection: {
        disableCache: true,
    },
});

configureWunderGraphApplication({
    // compose the APIs into a unified WunderGraph API
    apis: [api],
    security: {
        enableGraphQLEndpoint: true,
      }
  });
```

### Environment & setup

- OS: Ubuntu 20.04.5 LTS
- Go version: 1.18
- Database: PostgreSQL 15
- Node.js version: v18.15.0

### WunderCtl Version

```
Version: 0.138.0
Commit: c1286a7a51dbceb263e6b17f00a180a3b3e9d7e2
Date: 2023-04-04T12:04:53Z
BuiltBy: ci
```

**Comment by jensneuse** [comment]
Can you provide the schema to be able to reproduce the issue?

**Comment by rverma-nsl** [comment]
The problem is basically coming from https://github.com/wundergraph/wundergraph/blob/7977c79dc5dc9484c2412622828ac3a1380422fd/pkg/datasources/database/engine.go#L278

The whole schema is loaded to an environment variable here which is causing the issue.
As a workaround we are storing the large schema to a temporary file and passing the file as argument to prisma

```
func (e *Engine) StartQueryEngine(schemaFilePath string) error {

	err := e.ensurePrisma()
	if err != nil {
		return err
	}

	freePort, err := freeport.GetFreePort()
	if err != nil {
		return err
	}
	port := strconv.Itoa(freePort)
	ctx, cancel := context.WithCancel(context.Background())
	e.cancel = cancel
	e.cmd = exec.CommandContext(ctx, e.queryEnginePath, "-p", port, "--enable-raw-queries")

	// ensure that prisma starts with the dir set to the .wundergraph directory
	e.cmd.Dir = e.wundergraphDir

	e.cmd.Env = append(e.cmd.Env, os.Environ()...)
	
	// replace PRISMA_DML with PRISMA_DML_PATH
	if schemaFilePath != "" {
		e.cmd.Env = append(e.cmd.Env, "PRISMA_DML_PATH="+schemaFilePath)
	}

	e.cmd.Stdout = os.Stdout
	e.cmd.Stderr = os.Stderr
	e.url = "http://localhost:" + port
	if err := e.cmd.Start(); err != nil {
		e.StopQueryEngine()
		return err
	}
	return nil
}
```
Although its slightly twisted since the introspect method interface is expecting a schema and  with this introspection in prisma will happen like

introspectprismaFile(input: file) -> instrospect(input: prismaSchema) -> instrospectDatabase(prismaSchema > tmpFile);
We are basically rewriting the inputfile here.

[large.txt](https://github.com/wundergraph/wundergraph/files/11590149/large.txt)

**Comment by GandiYaswanthSai** [comment]
Hi @jensneuse , Please the below attached schema.
[schema.zip](https://github.com/wundergraph/wundergraph/files/11590766/schema.zip)

**Comment by rjackson** [comment]
@Slickstef11 I think this one has already been fixed in #1012?

**Comment by fiam** [comment]
Fixed by  #1012
